### DISCUSSION TITLE
how do i put Streets of Rage Remake on Adam Image?

### DISCUSSION THREAD

**Discussion by ashleyseare** [eduardofilo/RG350_adam_image#275] [Ports]
### What happened?

how do i put Streets of Rage Remake on Adam Image?.kernel version 5.17.0

### Device

RG350P

### Version

v2.0

**Comment by eduardofilo** [comment]
Check this:

https://github.com/eduardofilo/RG350_adam_image/wiki/En:-3.-Content-installation#ports
